### PR TITLE
Issue#1408: Log accurate error messages when devname is invalid

### PR DESCRIPTION
**Pull request by shuangzi52**
This pull request implement the following logic:
When we start observer by specifying dev name with -i, and the dev name is invalid, the accurate error messages will be write into observer.log

Issue: https://github.com/oceanbase/oceanbase/issues/1408

### CODE REVIEW

**Comment by a1iive** [comment]
Thanks for your commit! Can you post the results before and after the modification?

File: deps/oblib/src/lib/net/ob_net_util.cpp [hunk 1]
```diff
@@ -77,6 +77,13 @@ int ObNetUtil::get_local_addr_ipv6(const char *dev_name, char *ipv6, int len,
       } // if end
       ifa_tmp = ifa_tmp->ifa_next;
     } // while end
+
+    // if dev_name is invalid, than level will keep its initial value(-1)
```
Comment (hnwyllmm) [inline deps/oblib/src/lib/net/ob_net_util.cpp:81]: than -> then?

Comment (shuangzi52) [reply deps/oblib/src/lib/net/ob_net_util.cpp:81]: Yes, it should be [then].

**Comment by shuangzi52** [comment]
> Thanks for your commit! Can you post the results before and after the modification?

After the modification, there is an extra error message like this:

[2023-05-30 05:49:27.011010] WDIAG [LIB] get_local_addr_ipv4 (ob_net_util.cpp:125) [135017][observer][T0][Y0-0000000000000000-0-0] [lt=13][errcode=-4002] invalid devname specified by -i(devname="test_dev", info="you may list devnames by shell command: ifconfig")

File: deps/oblib/src/lib/net/ob_net_util.cpp (hunk 1 above)
Comment (shuangzi52) [reply deps/oblib/src/lib/net/ob_net_util.cpp]: > than -> then?

I have fixed the spelling error by changing [than] to [then].

**Review by a1iive — COMMENTED** [review]
LGTM

**Review by hnwyllmm — APPROVED** [review]
LGTM

**Comment by LINxiansheng** [comment]
Thanks for your contribution.

**Pull request merged**